tiny_dnn/util/parallel_for.h

```diff
@@ -31,6 +31,10 @@
 #include <thread>
 #endif
 
+#if defined(CNN_USE_GCD) && !defined(CNN_SINGLE_THREAD)
+#include <dispatch/dispatch.h>
+#endif
```

Comment:
`#if defined(CNN_USE_TBB)` then GCD version of `parallel_for` function will not appear after cpp preprocessing. That's alright but even in that case `<dispatch/dispatch.h>` header will be included and it will help consume tiny amount of electricity.